app/views/landowners/index.html.erb

```diff
@@ -0,0 +1 @@
+<%= react_component('LandownerPage', {word: "asf"}) %>

```

Comment:
What's this prop for?

Reply:
Oh good catch, its not really doing anything. Was there for testing something.